client/src/pages/CreateProfilePage/CreateProfilePage.css

```diff
@@ -0,0 +1,44 @@
+.create-profile-wrapper-div {
+  display: flexbox;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: white;
+  padding-left: 20%;
```

Comment:
this padding is pushing the content to the right when the window is sized down

Reply:
Sorry, I tried super hard but centering the items without using padding was really really hard